src/components/Proposals/ProposalCard.js

```diff
@@ -69,12 +79,14 @@ function ProposalCard({ vote, onProposalClick }) {
           margin-bottom: ${1 * GU}px;
         `}
       >
-        <AppBadge
-          label={target.name ? target.name : target.address}
-          appAddress={target.address}
-          iconSrc={target.icon ? target.icon : ''}
-          badgeOnly
-        />
+        {emptyScript ? (
+          <DefaultAppBadge />
```

Comment:
This was based on the previous version and is fallback disputable app badge. @delfipolito Are we sure this should be the default app when there is no description available to process?

Reply:
Yes :)

Reply:
Gotcha!